README.md

```diff
@@ -0,0 +1,80 @@
+  * [destinations](#destinations)
+  * [groups](#groups)
+  * [group_sources](#group_sources)
+  * [host_sources](#host_sources)
+* [Tests](#tests)
+* [Deploying](#deploying)
+
+## Customizations
+There are a few customization points for interferon that can be implemented in the alerts repository without having to commit changes to interferon itself. With the exception of `config.yml` the other entries in this list are all directories.
+
+### config.yml
+The `config.yml` file contains settings for interferon including which components to enable. It has the following configurable settings that can be set as YAML mapping keys:
+
+* **`verbose_logging`**: boolean value `true` or `false` which sets the logging verbosity for interferon.
+* **`alerts_repo_path`**: string value of the directory to look for the `alerts` directory and any additional directories for `destinations`, `group_sources`, `host_sources` modules.
+* **`destinations`** / **`group_sources`** / **`host_sources`**: list of maps containing the following keys.
+  * **`type`**: string value of the module to load for the given type (e.g. "filesystem" will look for `filesystem.rb` in the directory named as the current parent key in the `alerts_repo_path`).
+  * **`enabled`**: boolean value `true` or `false` whether or not to use a particular module.
+  * **`options`**: map of additional options to pass to the module.
+
+### alerts
+You can look at the example files in `alerts/` to get a taste of the syntax. You will need the following fields in your alerts file:
+
+* **`name`**: this will be the primary key of your alert, and will show up in emails and in the UI. Only one alert of a given name will be created, so if your alert name is not using some part of the `@hostinfo` object to make it unique, you will end up with a single alert.
+* **`message`**: the body of the alert that will be set to the destinations (currently only Datadog).
+* **`applies`**: determines which hosts the alert applies to; putting `true` will end up creating as many alerts as the number of distinct `name`s your alert expands into. Interferon will iterate through all the hosts generated by `host_sources` and test them agains the expression here.
+* **`notify.people`**: an array of the people who will be receiving the alert.
+* **`notify.groups`**: an array of groups who will be receiving the alert. The group mapping will be based on the output of `group_sources`. The `filesystem` group_source module will read the groups in the `groups` directory.
+* **`metric`**: The Datadog metric alert query.
+* **`silenced`**: The Datadog silenced setting.
+* **`timeout`**: The Datadog timeout setting in seconds (gets converted to `timeout_h`).
+
+### destinations
+This directory contains additional modules for `destinations`.
+
+### groups
+We only have a single group_source called `filesystem`, which reads groups in YAML files from the filesystem (specifically this `groups`). Each file contains the following keys:
```

Comment:
maybe:
```
This is the default location where the the built-in `filesystem` `group_source` reads it's YAML files.
If you don't want to store groups as YAML files, you can remove this directory, and also remove the `filesystem` group source from your `config.yml`.
Otherwise, you may configure groups (lists of people) here.
Each file contains...
```